Mathlib/Algebra/Group/Defs.lean

```diff
@@ -83,17 +83,17 @@ infixl:65 " +ᵥ " => HVAdd.hVAdd
 infixl:65 " -ᵥ " => VSub.vsub
 infixr:73 " • " => HSMul.hSMul
 
-attribute [to_additive] Mul Div HMul instHMul HDiv instHDiv HSMul
+attribute [to_additive existing] Mul Div HMul instHMul HDiv instHDiv HSMul
 attribute [to_additive (reorder := 1) SMul] Pow
 attribute [to_additive (reorder := 1)] HPow
-attribute [to_additive (reorder := 1 5) hSMul] HPow.hPow
-attribute [to_additive (reorder := 1 4) smul] Pow.pow
+attribute [to_additive existing (reorder := 1 5) hSMul] HPow.hPow
+attribute [to_additive (reorder := 1 4) existing smul] Pow.pow
```

Comment:
should the order of these clauses be standardized (line 89 versus line 90)?

Reply:
oh yes, my plan was to always put it in front of the configuration. I think I was testing whether it works after as well (I don't want to enforce an order in the attribute syntax - unless you think that is better). I'll fix it when I go through the library.